spec/controllers/api/posts_controller_spec.rb

```diff
@@ -2,6 +2,15 @@
 
 describe Api::PostsController do
   let(:current_skater_mock) { double('mock_current_skater', id: 1) }
+  let(:response_body) { JSON.parse(response.body) }
+
+  let(:mock_post) do
```

Comment:
Move this let and mock_posts let into the describe '#index' block, since they are only used inside that block